client/app/telecom/pack/xdsl/tasks/pack-xdsl-tasks.controller.js

```diff
@@ -18,9 +14,11 @@ angular.module('managerApp')
     };
 
     self.getTasks = function () {
-      OvhApiXdsl.v6().getTasks({ xdslId: $stateParams.serviceName }).$promise.then(
+      self.isLoading = true;
+      return OvhApiXdsl.v6().getTasks({ xdslId: $stateParams.serviceName }).$promise.then(
         (taskIds) => {
           self.taskIds = taskIds.map(taskId => ({ id: taskId }));
+          self.isLoading = false;
```

Comment:
This should probably go in a `finally` ? It will keep loading if there is an error